tests/integration/test_base.py

```diff
@@ -384,12 +387,20 @@ def assert_placed_item(self, item_value, zone_title, assessment_mode=False):
             self.assertDraggable(item_value)
             self.assertEqual(item.get_attribute('class'), 'option')
             self.assertEqual(item.get_attribute('tabindex'), '0')
-            self.assertEqual(item_description.text, 'Placed in: {}'.format(zone_title))
+            description = 'Placed in: {}'
         else:
             self.assertNotDraggable(item_value)
             self.assertEqual(item.get_attribute('class'), 'option fade')
             self.assertIsNone(item.get_attribute('tabindex'))
-            self.assertEqual(item_description.text, 'Correctly placed in: {}'.format(zone_title))
+            description = 'Correctly placed in: {}'
+
+        # Items with multiple possible drop zones could be placed correctly in
+        # one of several.  If zone_title is a list, check that it's in any one
+        # of the given zones.
```

Comment:
@arbrandes 

> ... check that it's in any one of the given zones.

Sorry for not noticing this earlier, but this part of the sentence is ambiguous: It's not clear what "it" is referring to. How about this:

> An item with multiple drop zones could be located in any one of these zones. In that case, `zone_title` will be a list, and we need to check whether the zone info in the description of the item matches any of the zones in that list.